src/containers/pages/OrganizationViews/OrganizationListView/index.tsx

```diff
@@ -112,11 +116,13 @@ const OrganizationListView = (props: OrgsListViewPropsType) => {
   }, []);
 
   // break early if organizations are absent
-  const isLoading = organizations.length < 1;
+  const isLoading = organizations.length < 1 && !props.name;
```

Comment:
Does this accurately present the state of "is loading"?

Reply:
@moshthepitt This is  a stand in fix that will allow the user to filter organization by any other title if the one they have filtered by; yields no organizations. I dd not intend for it to say anything about the loading state,.. since there is already a WIP pr #822 that will fix the whole loading state isue.